### PR TITLE
Content: Add Article for Dashboard 0.7

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

- Details the latest features & improvements added into `0.7.x` releases of Dashboard 2.0
- Focus on Migration Guide & update to `@flowfuse`
- Chart updates including key mapping & multiple series definitions

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes

### CODE REVIEW

**Comment by joepavitt** [comment]
Deploy Preview: https://deploy-preview-1311--flowforge-website.netlify.app/blog/2023/11/dashboard-0-7/

File: src/blog/2023/11/dashboard-0-7.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+We fully appreciate that the migration path is not yet complete, and we that we are missing some features and properties, but please know that we are working hard to ensure that as many of the features from Dashboard 1.0 are available in Dashboard 2.0. We will be updating this guide as we progress. 
+
+We will also be adding "event" and "dynamic properties" sections to the guide, to detail how you can update and control elements via runtime messages (e.g. dynamically change the label of a button), and how this differs (if at all) from Dashboard 1.0.
+
+Whilst we aren't quite there yet, this guide offers a comprehensive breakdown on our progress in backporting all of the properties from Dashboard 1.0
+
+#### Automated Script
+
+An ambitious plan that we have is to also provide a [Migration Script](https://github.com/FlowFuse/node-red-dashboard/issues/261). Any feedback, ideas or concerns are most welcome as comments on the issue.
```
Comment (ZJvandeWeg) [inline src/blog/2023/11/dashboard-0-7.md:68]: While I get this, I think if you're not going to get around to previous features this should not be mentioned until it comes to fruition?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/blog/2023/11/dashboard-0-7.md (hunk 1 above)
Comment (joepavitt) [reply src/blog/2023/11/dashboard-0-7.md:68]: Yeah, it's just been asked for by _so many_ people, I feel it has to be mentioned to save the guaranteed bombardment of questions asking for it

**Review by ZJvandeWeg — APPROVED** [review]

**Comment by ZJvandeWeg** [comment]
@Yndira-FlowForge Could you post this content to the social channels?

**Pull request merged**